enterprise/backend/src/metabase_enterprise/serialization/cmd.clj

```diff
@@ -142,9 +136,11 @@
                             :error_count   (count (:errors report))
                             :success       (nil? @err)
                             :error_message (some-> @err str)})
-    (when @err
-      (throw @err))
-    imported))
+    (if @err
+      (do
+        (serdes/log-stripped-error "Error during deserialization" @err)
+        (throw (ex-info (ex-message @err) {:cmd/exit true})))
+      imported)))
```

Comment:
I prefer the less nested `when` 🤷

Reply:
ah right it's now throw, I returned before; let me change that